src/core/reference/include/ngraph/runtime/reference/gelu.hpp

```diff
@@ -15,14 +15,14 @@ template <typename T>
 void gelu(const T* arg, T* out, op::GeluApproximationMode mode, size_t count) {
     if (mode == op::GeluApproximationMode::ERF) {
         for (size_t i = 0; i < count; i++) {
-            out[i] = 0.5 * arg[i] * (1 + erf(arg[i] / std::sqrt(2.0)));
+            out[i] = static_cast<T>((0.5 * arg[i] * (1 + erf(arg[i] / std::sqrt(2.0)))));
```

Comment:
maybe if floats? Otherwise it's very slow on ARM

Reply:
It is reference ops, do need concern the performance?